docs/tutorial/cpp/src/getting_started.md

```diff
@@ -1,88 +1,82 @@
 <!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+
 # Getting Started
 
-In this tutorial, we use C++ as the host programming language. We also support other programming languages like
+This tutorial uses C++ as the host programming language. Slint also supports other programming languages like
 [Rust](https://slint.dev/docs/rust/slint/) or [JavaScript](https://slint.dev/docs/node/).
 
-You will need a development environment that can compile C++20, [CMake 3.21](https://cmake.org/download/),
+Slint has an application template you can use to create a project with dependencies already set up that follows recommended best practices.
+
+Before using the template, you need a C++ compiler that supports C++ 20 and to install [CMake](https://cmake.org/download/) 3.21 or newer.
+
+Clone or download template repository:
+
+```sh
+git clone https://github.com/slint-ui/slint-cpp-template memory
+cd memory
+```
+
+<!-- You will need a development environment that can compile C++20, [CMake 3.21](https://cmake.org/download/),
 and we recommend [Ninja](https://ninja-build.org) for `-GNinja`.
 We don't provide binaries of Slint yet, so we will use the CMake integration that will automatically build
 the tools and library from source. Since it's implemented in the Rust programming language, this means that
 you also need to install a Rust compiler (1.70 or newer). You can easily install a Rust compiler
 following the instruction from [the Rust website](https://www.rust-lang.org/learn/get-started).
-We're going to use `cmake`'s builtin FetchContent module to fetch the source code of Slint.
-
-In a new directory, we create a new `CMakeLists.txt` file.
-
-```cmake
-# CMakeLists.txt
-cmake_minimum_required(VERSION 3.21)
-project(memory LANGUAGES CXX)
-
-include(FetchContent)
-FetchContent_Declare(
-    Slint
-    GIT_REPOSITORY https://github.com/slint-ui/slint.git
-    # `release/1` will auto-upgrade to the latest Slint >= 1.0.0 and < 2.0.0
-    # `release/1.0` will auto-upgrade to the latest Slint >= 1.0.0 and < 1.1.0
-    GIT_TAG release/1
-    SOURCE_SUBDIR api/cpp
-)
-FetchContent_MakeAvailable(Slint)
-
-add_executable(memory_game main.cpp)
-target_link_libraries(memory_game PRIVATE Slint::Slint)
-slint_target_sources(memory_game memory.slint)
-# On Windows, copy the Slint DLL next to the application binary so that it's found.
-if (WIN32)
-    add_custom_command(TARGET memory_game POST_BUILD COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_RUNTIME_DLLS:memory_game> $<TARGET_FILE_DIR:memory_game> COMMAND_EXPAND_LISTS)
-endif()
+We're going to use `cmake`'s builtin FetchContent module to fetch the source code of Slint. -->
+
+Change the references to `my_application` in the `CMakeLists.txt` file to `memory`
```

Comment:
I wonder if the renaming is worth it though - it makes it a little error prone since the instructions just say "change the references".

Reply:
@tronical What do you mean? The template project has everything set with different file names, maybe I'm misunderstanding something about C++?

Reply:
@tronical Ahhh, I see. Hmm, I did have to delete build artefacts every time, but I wasn't sure if that was a normal thing to have to do with C++ anyway 😅 . I don't mind, really. If you think it's likely less reliable not to rename it, then I can leave it. I'd rather people had a smooth first Slint experience than get bogged down in C++ issues.

Reply:
Yep, that's exactly my worry. I'm leaning towards taking the safe route for now (and not renaming) - the lower the risk the higher the chance of a smooth first experience.